### ISSUE TITLE
Allow ServiceBinding generate Secret into multiple namespaces

### ISSUE THREAD

**Issue by pre** [SAP/sap-btp-service-operator#361]
In a cluster of microservices, many ServiceBinding resources are shared between multiple applications deployed in different Kubernetes namespaces. 

We're currently using an external component which watches the `Secret` resources generated by sap-btp-service-operator and copies these `Secrets` to multiple namespaces. The namespaces are defined in an annotation to the `ServiceBinding` such as 

```yaml
apiVersion: services.cloud.sap.com/v1alpha1
kind: ServiceBinding
metadata:
  name: example-binding
  namespace: integration
  annotations:
    klacks.io/include-namespaces: >-
      application-1,
      another-application,
      third-application,
      some-other-application,
      also-this-application
    klacks.io/synchronizer: "yes"
spec:
  serviceInstanceName: example-ias-instance
  secretName: example-ias-binding
  parameters:
    credential-type: X509_GENERATED
    key-length: 2048
    validity: 60
    validity-type: DAYS
    app-identifier: example-app-identifier
  credentialsRotationPolicy:
    enabled: true
    rotationFrequency: 336h
    rotatedBindingTTL: 336h
```

There was a recent Pull Request for creating the ServiceBinding in a different namespace than the ServiceInstance. This is nice, however, it doesn't really solve this problem because it only allows a single namespace. 
* https://github.com/SAP/sap-btp-service-operator/pull/321

With multiple `ServiceBinding`  resources, we'd either need to duplicate the `spec`  for each of them or use templating. , Templating the `spec` has risks of its own when propagating changes over different namespaces. 

TL;DR Having a single `spec` with a `Secret` copied to multiple namespaces feels more maintainable than creating multiple `ServiceBinding` resources of which each requires a `spec` of its own. 

The code for the external component, which synchronizes the generated `Secret` over namespaces defined in the `ServiceBinding`'s annotations, is here https://github.tools.sap/AppGyver/appgyver-k8s-synchronizer - It was specifically built for this purpose to supplement sap-btp-service-operator. Maybe you could even use it as a reference?

**Comment by loewenstein** [comment]
I wonder whether they should share a single binding secret though and the request here should rather be allowing to create ServiceBinding resources in multiple namespaces that bind to a single ServiceInstance. ???

**Comment by pre** [comment]
@loewenstein Yes, they need to share the same `ServiceBinding`. This is how BTP works internally in relation to our BTP Subscription Tree.
